binder/nanoevents.ipynb

```diff
@@ -126,7 +152,29 @@
    "cell_type": "code",
    "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\u001b[0;31mType:\u001b[0m            Array\n",
+       "\u001b[0;31mString form:\u001b[0m     dask.awkward<id1, npartitions=1>\n",
+       "\u001b[0;31mFile:\u001b[0m            /opt/homebrew/lib/python3.11/site-packages/dask_awkward/lib/core.py\n",
+       "\u001b[0;31mDocstring:\u001b[0m       id of first parton\n",
+       "\u001b[0;31mClass docstring:\u001b[0m\n",
+       "Partitioned, lazy, and parallel Awkward Array Dask collection.\n",
+       "\n",
+       "The class constructor is not intended for users. Instead use\n",
+       "factory functions like :py:func:`~dask_awkward.from_parquet`,\n",
+       ":py:func:`~dask_awkward.from_json`, etc.\n",
+       "\n",
+       "Within dask-awkward the ``new_array_object`` factory function is\n",
+       "used for creating new instances.\n"
+      ]
+     },
```

Comment:
Thanks for the review!

One minor question and then this should be ready to go.

This part is supposed to output information about `Generator.id1` -
```
Type:            Array
String form:     [1, -1, -1, 21, 21, 4, 2, -2, 2, 1, 3, 1, ... -1, -1, 1, -2, 2, 1, 2, -2, -1, 2, 1]
Length:          40
File:            ~/src/awkward-1.0/awkward1/highlevel.py
Docstring:       id of first parton
Class docstring: ...
```
but instead, it outputs the docs of the Dask compatible Array class. Is there a new way to display or access the information of a `TBranch`?

Reply:
I don't think so since we never have access to the opened root file, just dask mapping to it, so the dask-array for the class docstring is actually correct (if confusing).

Reply:
Oh, I see, thanks! This should be ready then.